themes/agnoster.zsh-theme

```diff
@@ -198,6 +198,19 @@ prompt_virtualenv() {
   fi
 }
 
+# Anaconda: current working conda env
+# To replace the conda modification of PS1 do `conda config --set changeps1 False`
+# or add 'changeps1: False' to your .condarc file.
+prompt_conda() {
+  local conda_env="$CONDA_DEFAULT_ENV"
+  if [[ -n $conda_env ]]; then
+    local conda_changeps1="`conda config --show changeps1`" 
+    if [[ $conda_changeps1 == "changeps1: False" && $conda_env != "base" ]]; then
```

Comment:
You basically invoke `conda` every time `PS1` is evaluated. This will incur a heavy performance penalty on the user.
Instead, you could check the `CONDA_PROMPT_MODIFIER` env variable.

When `changeps1` is set to `true`, this var will be `(<env name>)`.
When it is `false`, the var will be empty.
In addition, I don't see a reason to hide the `base` env. I would really like to know if I'm activated or not.

```suggestion 
    if [[ -z $CONDA_PROMPT_MODIFIER ]]; then
```